src/draive/types/audio.py

```diff
@@ -11,12 +11,14 @@ class AudioURLContent(DataModel):
     mime_type: str | None = None
     audio_url: str
     audio_transcription: str | None = None
+    meta: dict[str, str | float | int | bool | None] | None = None
```

Comment:
There's no possibility to pass a nested dict?

Reply:
no and that is intentional to keep it concise - I don't want to put there a ton of information, the primary purpose of MultimodalContent is to make abstraction over multimodal content passed to or from LLMs, meta will be mostly ignored in that case, it is intended for additional processing and custom labeling of parts of the content i.e. to mark text section as a header and another one as a content